api/src/main/java/pl/bestguilds/api/user/UserStatistic.java

```diff
@@ -0,0 +1,31 @@
+package pl.bestguilds.api.user;
+
+public interface UserStatistic {
+
+  int getPoints();
```

Comment:
Hmm... why not be more accurate here and store points in floats/doubles and then just round them when they are shown for players?